_posts/2020-01-07-quarkus-eclipse-microprofile-3-2-compatible.adoc

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: Quarkus is Eclipse MicroProfile 3.2 compatible!
+date: 2020-01-07
+tags: quarkus microprofile
+author: kenfinnigan
+---
+
+With the recent announcement that Quarkus 1.1 is now available,
+and after a tremendous effort from all the contributors and community,
+we’re also excited to announce that Quarkus 1.1 is Eclipse MicroProfile 3.2 compatible!
+https://projects.eclipse.org/projects/technology.microprofile/releases/microprofile-3.2[MicroProfile 3.2] was released this week,
```

Comment:
MP 3.2 was released back in November AFAIK (https://github.com/eclipse/microprofile/releases/tag/3.2) so this "last week" reference doesn't look right to me in a post dated 2020-01-07